docs/nebula-design-concepts.md

```diff
@@ -0,0 +1,216 @@
+# Data modeling and design concept
+------
+**Nebula Graph** is comprised of four components: storage layer, metadata service, compute layer and client console. In this documentation, we'll discuss data modeling and architecture of Nebula Graph.
+
+---
+## Directed property graph
+
+Modeling data with directed property graph, Nebula Graph is very easy to understand.
```

Comment:
Entities and connections are modeled as directed property graph in Nebula Graph.

Reply:
Modeling data with directed property graph, Nebula Graph is very easy to understand,
i.e. Nebula Graph consists of two graph elements: vertex and edge.

There are explanations on entities and connections after the sentence.